ci-operator/config/openshift/loki/openshift-loki-release-5.8.yaml

```diff
@@ -0,0 +1,153 @@
+base_images:
+  base:
+    name: "4.14"
+    namespace: ocp
+    tag: base
+  lokistack-gateway:
+    name: mirror-5.y
+    namespace: logging
+    tag: lokistack-gateway
+  ocp_builder_rhel-9-golang-1.20-openshift-4.14:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.20-openshift-4.14
+  opa-openshift:
+    name: mirror-5.y
+    namespace: logging
+    tag: opa-openshift
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
+images:
+- dockerfile_path: Dockerfile.ocp
+  from: base
+  inputs:
+    ocp_builder_rhel-9-golang-1.20-openshift-4.14:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.20-openshift-4.14
+  to: loki
+- dockerfile_path: Dockerfile.promtail.ocp
+  from: base
+  inputs:
+    ocp_builder_rhel-9-golang-1.20-openshift-4.14:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.20-openshift-4.14
+  to: promtail
```

Comment:
We are not building `loki` and `promtail` on the other release branches. I don't think we want to start doing this?

```suggestion
```